rocketpool-cli/commands/pdao/proposals.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/rocket-pool/rocketpool-go/v2/types"
 	"github.com/urfave/cli/v2"
 
+	utilsMath "github.com/rocket-pool/node-manager-core/utils/math"
```

Comment:
golang package convention is, for whatever reason, c-style, eg `utilsmath` and `utilsstr`:

https://go.dev/blog/package-names

![image](https://github.com/rocket-pool/smartnode/assets/116244/7de38a94-8cb6-4c62-9a88-f11158a846b1)